src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -337,335 +359,345 @@ public void shouldSendAppointeeNotificationsForAnEventForAnOralOrPaperHearingAnd
-                                SYA_APPEAL_CREATED_NOTIFICATION,
-                                "oral",
-                                Arrays.asList("01293b93-b23e-40a3-ad78-2c6cd01cd21c", "652753bf-59b4-46eb-9c24-bd762338a098"),
-                                Arrays.asList("f41222ef-c05c-4682-9634-6b034a166368", "0e44927e-168b-4510-ac57-6932fda7aec1"),
-                                "yes",
-                                "yes",
-                                "yes",
-                                "yes",
-                                "2",
-                                "2"
-                    },
-                    new Object[]{
-                                SYA_APPEAL_CREATED_NOTIFICATION,
-                                "paper",
-                                Collections.singletonList("652753bf-59b4-46eb-9c24-bd762338a098"),
-                                Arrays.asList("f41222ef-c05c-4682-9634-6b034a166368", "0e44927e-168b-4510-ac57-6932fda7aec1"),
-                                "no",
-                                "yes",
-                                "yes",
-                                "yes",
-                                "1",
-                                "2"
-                    },
-                    new Object[]{
-                                POSTPONEMENT_NOTIFICATION,
-                                "paper",
-                                Arrays.asList("08959288-e09a-472d-80b8-af79bfcbb437", "0a48bd48-f79c-4863-b6e3-e8fa69019c34"),
-                                Collections.EMPTY_LIST,
-                                "yes",
-                                "no",
-                                "yes",
-                                "no",
-                                "2",
-                                "0"
-                    },
-                    new Object[]{
-                                POSTPONEMENT_NOTIFICATION,
-                                "oral",
-                                Arrays.asList("08959288-e09a-472d-80b8-af79bfcbb437", "0a48bd48-f79c-4863-b6e3-e8fa69019c34"),
-                                Collections.EMPTY_LIST,
-                                "yes",
-                                "no",
-                                "Yes",
-                                "no",
-                                "2",
-                                "0"
-                    },
-                    new Object[]{
-                                POSTPONEMENT_NOTIFICATION,
-                                "paper",
-                                Collections.singletonList("0a48bd48-f79c-4863-b6e3-e8fa69019c34"),
-                                Collections.EMPTY_LIST,
-                                "no",
-                                "no",
-                                "yes",
-                                "no",
-                                "1",
-                                "0"
-                    }
+                new Object[]{
```

Comment:
I see these white spaces going back and forth. Is there a common config file we can share so that people's IDEs can autoformat consistently?

Reply:
addressed in slack

Reply:
This is true :-)